README.md

```diff
@@ -11,6 +11,8 @@
 
 This package [has been donated](https://github.com/facebook/jest/issues/1679) to [Jest](https://github.com/facebook/jest). This means that all future development of `expect` v21+ will take place at [facebook/jest](https://github.com/facebook/jest).
 
+See [facebook/jest/packages/expect](https://github.com/facebook/jest/tree/master/packages/expect) for the source and documentation of the current NPM package.
```

Comment:
This is the same thing the previous paragraph communicates; altho I suppose we could update the `facebook/jest` link to point directly at the expect package for v21+?